### PR TITLE
feat(preview): update freemium preview sidebar for upsell

### PR DESCRIPTION
**Pull request by wlyhdg**
present the winning treatment preview sidebar for when upgrading is applicable
Screen Shot 2022-06-07 at 12 03 55 PM

### CODE REVIEW

File: src/elements/content-sidebar/versions/VersionsSidebarContainer.js [hunk 1]
```diff
@@ -37,14 +37,8 @@ type Props = {
     onVersionPromote: VersionActionCallback,
     onVersionRestore: VersionActionCallback,
     parentName: string,
+    showUpsellExperience: boolean,
```
Comment (jstoffan) [inline src/elements/content-sidebar/versions/VersionsSidebarContainer.js:40]: Is this still needed or can we use the existence of `onUpgradeClick` to determine if the static sidebar should be shown?

**Review by jstoffan — APPROVED** [review]
Thanks for simplifying this code path. In the future, it can be helpful to include a screenshot and description so folks can quickly assess the new behaviors.

File: src/elements/content-sidebar/versions/VersionsSidebarContainer.js (hunk 1 above)
Comment (wlyhdg) [reply src/elements/content-sidebar/versions/VersionsSidebarContainer.js:40]: @jstoffan onUpgradeClick is being passed as `onUpgradeClick: showUpsellExperience && onUpgradeClick` so I do think that our safeguard would still apply. this is a good simplification but do you think this syntax is clear enough for a user (not having the showUpsellExperience explicitly and just using onUpgradeClick)? if so can make the updates!

**Comment by wlyhdg** [comment]
> Thanks for simplifying this code path. In the future, it can be helpful to include a screenshot and description so folks can quickly assess the new behaviors.

thank you, will do! going to add with a few updates

File: src/elements/content-sidebar/versions/VersionsSidebarContainer.js (hunk 1 above)
Comment (jstoffan) [reply src/elements/content-sidebar/versions/VersionsSidebarContainer.js:40]: Yeah, I think it should be fine. We're likely the only ones using these props and `onUpgradeClick` isn't used for anything else currently. Best to keep the API as small as possible, I think.

**Review by jstoffan — DISMISSED** [review]

**Review by jstoffan — APPROVED** [review]

**Review by rdacanay — DISMISSED** [review]

**Review by jstoffan — APPROVED** [review]

**Pull request merged**